suites/pacific/rbd/tier-2_rbd_mirroring.yaml

```diff
@@ -0,0 +1,83 @@
+# Tier2: Extended RBD testing
```

Comment:
We recommend to have the file name as `tier-2_rbd_mirror_test-basic-features-of-mirroring.yaml`. This allows to get metadata information from the filename.

Reply:
done